webui/react/src/pages/F_ExpList/glide-table/columns.ts

```diff
@@ -125,81 +132,83 @@ export const getColumnDefs = ({
       kind: GridCellKind.Text,
     }),
     title: 'Description',
+    tooltip: () => undefined,
     width: columnWidths.description,
   },
   duration: {
     id: 'duration',
     isNumerical: true,
     renderer: (record: ExperimentItem) => ({
       allowOverlay: false,
-      data: `${getDurationInEnglish(record)}`,
-      displayData: `${getDurationInEnglish(record)}`,
+      data: getDurationInEnglish(record),
```

Comment:
suggestion: worth adding this helper to `utils/datetime.ts` to potentially reuse elsewhere?